clang/cmake/caches/Fuchsia-stage2.cmake

```diff
@@ -30,7 +30,6 @@ set(LLDB_ENABLE_CURSES OFF CACHE BOOL "")
 set(LLDB_ENABLE_LIBEDIT OFF CACHE BOOL "")
 
 if(WIN32)
-  set(LLVM_USE_CRT_RELEASE "MT" CACHE STRING "")
```

Comment:
This case here seems to lack a corresponding setting of `CMAKE_MSVC_RUNTIME_LIBRARY` - I guess we should add that to make this a proper no-op?

Reply:
It's been set at Line 57?